src/main/scala/uk/gov/hmrc/playcrosscompilation/AbstractPlayCrossCompilation.scala

```diff
@@ -78,20 +72,20 @@ abstract class AbstractPlayCrossCompilation(
     unmanagedResourceDirectories in Test += {
       (sourceDirectory in Test).value / playDir / "resources"
     },
-    scalaVersion := playScalaVersion(crossScalaVersions.value)(scalaVersion.value),
-    crossScalaVersions ~= playCrossScalaBuilds
+    scalaVersion := scalaVersion.value,
+    crossScalaVersions := crossScalaVersions.value
```

Comment:
Can we just remove this line, since we are not modifying `crossScalaVersions` now

Reply:
done.